Project/Source/AnimationController.h

```diff
@@ -8,6 +8,7 @@ class GameObject;
 class State;
 class AnimationController {
 public:
+	static int GetCurrentSample(const ResourceClip& clip, float& currentTime);
```

Comment:
I think this class should be a namespace instead of a class with all members static